x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.scss

```
@@ -0,0 +1,26 @@
+.appSearchResultHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.appSearchResultHeaderColumn {
+  display: flex;
+  flex-wrap: wrap;
+
+  &--left {
+    justify-content: flex-start;
+
+    & .appSearchResultHeaderItem {
+      margin-right: $euiSizeL;
+    }
+  }
+
+  &--right {
+    justify-content: flex-end;
+
+    & .appSearchResultHeaderItem + .appSearchResultHeaderItem {
```

Super appreciate the div cleanup change! ❤️ for basic nested selectors I have a slight preference to dropping the explicit `&` - not sure if it's significantly more readable, but it's what I (think?) most people are used to

```suggestion
    .appSearchResultHeaderItem {
      margin-right: $euiSizeL;
    }
  }

  &--right {
    justify-content: flex-end;

    .appSearchResultHeaderItem + .appSearchResultHeaderItem {
```

---

Ah, yes, of course. I just overlooked this, good call. 2ccd75c7647ac8754688f04e792c47337e35347a